pebbles/static/js/controllers/DashboardController.js

```diff
@@ -192,6 +192,10 @@ app.controller('DashboardController', ['$q', '$scope', '$routeParams', '$interva
         $scope.provision = function (blueprint) {
             instances.post({blueprint: blueprint.id}).then(function (response) {
                 $scope.updateInstanceList();
+                $timeout(function () {
+                    instances.get(response.id, {'check_provisioning': true}).then(function (response) {
```

Comment:
Since, we are sending emails to the users, this is a 'side-effect' and it should in the PATCH method.